tests/std/tests/P0220R1_string_view/test.cpp

```diff
@@ -1251,6 +1252,38 @@ static_assert(!is_constructible_v<string, nullptr_t>, "constructing string from
 static_assert(!is_assignable_v<string&, nullptr_t>, "assigning nullptr_t to string is prohibited");
 #endif // _HAS_CXX23
 
+// Also test that no C6510 warning
+struct char_wrapper {
+    char c;
+};
+
+template <>
+struct std::char_traits<char_wrapper> {
+    using char_type = char_wrapper;
+
+    static bool eq(char_wrapper lhs, char_wrapper rhs) {
+        return lhs.c == rhs.c;
+    }
+
+    static size_t length(const char_wrapper* a) {
+        static_assert(sizeof(char_wrapper) == 1, "strlen requires this");
+        return std::strlen(reinterpret_cast<const char*>(a));
+    }
+
+    static int compare(const char_wrapper* lhs, const char_wrapper* rhs, std::size_t count) {
+        return std::char_traits<char>::compare(

```

Comment:
Ditto "using `std`":
```suggestion
    static int compare(const char_wrapper* lhs, const char_wrapper* rhs, size_t count) {
        return char_traits<char>::compare(
```